transport/server.cc

```diff
@@ -157,31 +159,34 @@ cql_server::cql_server(distributed<service::storage_proxy>& proxy, distributed<c
 {
     namespace sm = seastar::metrics;
 
+    auto proto_label_instance = class_label("cql3");
+
```

Comment:
It's really sad that the protocol is named "cql", because that's really the name of the query language and causes confusion.

However, let's call it cql and not cql3 as this is the name used in Cassandra documentation. Or maybe even "cqlbinary" to distinguish it from the language (the docs reference it as "CQL BINARY PROTOCOL").

Note that the thrift protocol can process cql3 statements.